src/pages/Explore/Explore.tsx

```diff
@@ -25,7 +25,7 @@ class Explore extends React.Component<{}, ExploreState> {
   }
 
   render() {
-      const cards = this.state.modules.map((m, i) => <ModuleCard buttonLink={'/modules/' + m.id} module={m} key={i}/>);
+      const cards = this.state.modules.map((m, i) => <ModuleCard buttonLink={'/module/' + m.id} module={m} key={i}/>);
```

Comment:
In ModuleCard.tsx, the button to go to the module is hidden if not authenticated. The user should be able to go to the module, just not start it if they are not authenticated.